<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="3" ignored="0" total="9" passed="5" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[ContactJamess_1537 added Successfully]]>
    </line>
    <line>
      <![CDATA[CampaignGoGreen_691 added Successfully]]>
    </line>
    <line>
      <![CDATA[CampaignGoGreen_68 added Successfully]]>
    </line>
    <line>
      <![CDATA[CampaignGoGreen_1751 added Successfully]]>
    </line>
  </reporter-output>
  <suite started-at="2025-03-29T20:48:07 IST" name="Suite" finished-at="2025-03-29T20:50:00 IST" duration-ms="113263">
    <groups>
      <group name="SmokeTesting">
        <method signature="createProduct.createProductTest()[pri:0, instance:ProductTest.createProduct@1ff4931d]" name="createProductTest" class="ProductTest.createProduct"/>
      </group> <!-- SmokeTesting -->
      <group name="IntegrationTesting">
        <method signature="createContactWithCampaign.createContactWithCampaignTest()[pri:0, instance:contactTest.createContactWithCampaign@3a52dba3]" name="createContactWithCampaignTest" class="contactTest.createContactWithCampaign"/>
      </group> <!-- IntegrationTesting -->
    </groups>
    <test started-at="2025-03-29T20:48:07 IST" name="Test" finished-at="2025-03-29T20:50:00 IST" duration-ms="113263">
      <class name="CampaignTest.CreateCampaign">
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:CampaignTest.CreateCampaign@710f4dc7]" started-at="2025-03-29T20:49:01 IST" name="beforeClass" finished-at="2025-03-29T20:49:04 IST" duration-ms="2435" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:CampaignTest.CreateCampaign@710f4dc7]" started-at="2025-03-29T20:49:04 IST" name="beforeMethod" finished-at="2025-03-29T20:49:05 IST" duration-ms="1516" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="createCampaignWIthMandatoryFieldsTest()[pri:0, instance:CampaignTest.CreateCampaign@710f4dc7]" started-at="2025-03-29T20:49:05 IST" name="createCampaignWIthMandatoryFieldsTest" finished-at="2025-03-29T20:49:15 IST" duration-ms="9987" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[CampaignGoGreen_691 added Successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- createCampaignWIthMandatoryFieldsTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:CampaignTest.CreateCampaign@710f4dc7]" started-at="2025-03-29T20:49:15 IST" name="afterMethod" finished-at="2025-03-29T20:49:16 IST" duration-ms="780" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:CampaignTest.CreateCampaign@710f4dc7]" started-at="2025-03-29T20:49:16 IST" name="beforeMethod" finished-at="2025-03-29T20:49:17 IST" duration-ms="886" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="createCampaignWithAllFieldsTest()[pri:0, instance:CampaignTest.CreateCampaign@710f4dc7]" started-at="2025-03-29T20:49:17 IST" name="createCampaignWithAllFieldsTest" finished-at="2025-03-29T20:49:29 IST" duration-ms="12106" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[CampaignGoGreen_68 added Successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- createCampaignWithAllFieldsTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:CampaignTest.CreateCampaign@710f4dc7]" started-at="2025-03-29T20:49:29 IST" name="afterMethod" finished-at="2025-03-29T20:49:29 IST" duration-ms="349" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:CampaignTest.CreateCampaign@710f4dc7]" started-at="2025-03-29T20:49:29 IST" name="beforeMethod" finished-at="2025-03-29T20:49:30 IST" duration-ms="618" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="createCampaignWithCloseDateTest()[pri:0, instance:CampaignTest.CreateCampaign@710f4dc7]" started-at="2025-03-29T20:49:30 IST" name="createCampaignWithCloseDateTest" finished-at="2025-03-29T20:49:39 IST" duration-ms="8661" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[CampaignGoGreen_1751 added Successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- createCampaignWithCloseDateTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:CampaignTest.CreateCampaign@710f4dc7]" started-at="2025-03-29T20:49:39 IST" name="afterMethod" finished-at="2025-03-29T20:49:39 IST" duration-ms="345" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:CampaignTest.CreateCampaign@710f4dc7]" started-at="2025-03-29T20:49:39 IST" name="afterClass" finished-at="2025-03-29T20:49:39 IST" duration-ms="281" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- CampaignTest.CreateCampaign -->
      <class name="ProductTest.createProduct">
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:ProductTest.createProduct@1ff4931d]" started-at="2025-03-29T20:49:39 IST" name="beforeClass" finished-at="2025-03-29T20:49:41 IST" duration-ms="1395" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:ProductTest.createProduct@1ff4931d]" started-at="2025-03-29T20:49:41 IST" name="beforeMethod" finished-at="2025-03-29T20:49:42 IST" duration-ms="1205" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="createProductTest()[pri:0, instance:ProductTest.createProduct@1ff4931d]" started-at="2025-03-29T20:49:42 IST" name="createProductTest" finished-at="2025-03-29T20:49:59 IST" duration-ms="17205" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createProductTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:ProductTest.createProduct@1ff4931d]" started-at="2025-03-29T20:49:59 IST" name="afterMethod" finished-at="2025-03-29T20:50:00 IST" duration-ms="535" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:ProductTest.createProduct@1ff4931d]" started-at="2025-03-29T20:50:00 IST" name="afterClass" finished-at="2025-03-29T20:50:00 IST" duration-ms="284" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- ProductTest.createProduct -->
      <class name="LoginTest.Login">
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:LoginTest.Login@2925bf5b]" started-at="2025-03-29T20:48:31 IST" name="beforeClass" finished-at="2025-03-29T20:48:33 IST" duration-ms="1353" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:LoginTest.Login@2925bf5b]" started-at="2025-03-29T20:48:33 IST" name="beforeMethod" finished-at="2025-03-29T20:48:35 IST" duration-ms="2132" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method retried="true" signature="loginTest()[pri:0, instance:LoginTest.Login@2925bf5b]" started-at="2025-03-29T20:48:35 IST" name="loginTest" finished-at="2025-03-29T20:48:40 IST" duration-ms="5077" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Validation is failed expected [http://49.249.28.218:8098/dashbod] but found [http://49.249.28.218:8098/dashboard]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Validation is failed expected [http://49.249.28.218:8098/dashbod] but found [http://49.249.28.218:8098/dashboard]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at LoginTest.Login.loginTest(Login.java:91)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:LoginTest.Login@2925bf5b]" started-at="2025-03-29T20:48:40 IST" name="afterMethod" finished-at="2025-03-29T20:48:41 IST" duration-ms="789" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:LoginTest.Login@2925bf5b]" started-at="2025-03-29T20:48:41 IST" name="beforeMethod" finished-at="2025-03-29T20:48:41 IST" duration-ms="857" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method retried="true" signature="loginTest()[pri:0, instance:LoginTest.Login@2925bf5b]" started-at="2025-03-29T20:48:41 IST" name="loginTest" finished-at="2025-03-29T20:48:46 IST" duration-ms="4553" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Validation is failed expected [http://49.249.28.218:8098/dashbod] but found [http://49.249.28.218:8098/dashboard]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Validation is failed expected [http://49.249.28.218:8098/dashbod] but found [http://49.249.28.218:8098/dashboard]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at LoginTest.Login.loginTest(Login.java:91)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:LoginTest.Login@2925bf5b]" started-at="2025-03-29T20:48:46 IST" name="afterMethod" finished-at="2025-03-29T20:48:46 IST" duration-ms="386" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:LoginTest.Login@2925bf5b]" started-at="2025-03-29T20:48:46 IST" name="beforeMethod" finished-at="2025-03-29T20:48:47 IST" duration-ms="734" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method retried="true" signature="loginTest()[pri:0, instance:LoginTest.Login@2925bf5b]" started-at="2025-03-29T20:48:47 IST" name="loginTest" finished-at="2025-03-29T20:48:52 IST" duration-ms="4735" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Validation is failed expected [http://49.249.28.218:8098/dashbod] but found [http://49.249.28.218:8098/dashboard]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Validation is failed expected [http://49.249.28.218:8098/dashbod] but found [http://49.249.28.218:8098/dashboard]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at LoginTest.Login.loginTest(Login.java:91)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:LoginTest.Login@2925bf5b]" started-at="2025-03-29T20:48:52 IST" name="afterMethod" finished-at="2025-03-29T20:48:52 IST" duration-ms="490" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:LoginTest.Login@2925bf5b]" started-at="2025-03-29T20:48:52 IST" name="beforeMethod" finished-at="2025-03-29T20:48:53 IST" duration-ms="942" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="loginTest()[pri:0, instance:LoginTest.Login@2925bf5b]" started-at="2025-03-29T20:48:53 IST" name="loginTest" finished-at="2025-03-29T20:48:58 IST" duration-ms="4834" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Validation is failed expected [http://49.249.28.218:8098/dashbod] but found [http://49.249.28.218:8098/dashboard]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Validation is failed expected [http://49.249.28.218:8098/dashbod] but found [http://49.249.28.218:8098/dashboard]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at LoginTest.Login.loginTest(Login.java:91)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:LoginTest.Login@2925bf5b]" started-at="2025-03-29T20:48:59 IST" name="afterMethod" finished-at="2025-03-29T20:48:59 IST" duration-ms="615" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:LoginTest.Login@2925bf5b]" started-at="2025-03-29T20:48:59 IST" name="afterClass" finished-at="2025-03-29T20:49:01 IST" duration-ms="2110" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- LoginTest.Login -->
      <class name="contactTest.createContactWithCampaign">
        <test-method is-config="true" signature="beforeSuite()[pri:0, instance:contactTest.createContactWithCampaign@3a52dba3]" started-at="2025-03-29T20:48:06 IST" name="beforeSuite" finished-at="2025-03-29T20:48:07 IST" duration-ms="617" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:contactTest.createContactWithCampaign@3a52dba3]" started-at="2025-03-29T20:48:07 IST" name="beforeTest" finished-at="2025-03-29T20:48:07 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:contactTest.createContactWithCampaign@3a52dba3]" started-at="2025-03-29T20:48:07 IST" name="beforeClass" finished-at="2025-03-29T20:48:09 IST" duration-ms="2466" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:contactTest.createContactWithCampaign@3a52dba3]" started-at="2025-03-29T20:48:09 IST" name="beforeMethod" finished-at="2025-03-29T20:48:10 IST" duration-ms="1084" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="createContactWithCampaignTest()[pri:0, instance:contactTest.createContactWithCampaign@3a52dba3]" started-at="2025-03-29T20:48:10 IST" name="createContactWithCampaignTest" finished-at="2025-03-29T20:48:30 IST" duration-ms="19924" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[ContactJamess_1537 added Successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- createContactWithCampaignTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:contactTest.createContactWithCampaign@3a52dba3]" started-at="2025-03-29T20:48:30 IST" name="afterMethod" finished-at="2025-03-29T20:48:31 IST" duration-ms="821" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:contactTest.createContactWithCampaign@3a52dba3]" started-at="2025-03-29T20:48:31 IST" name="afterClass" finished-at="2025-03-29T20:48:31 IST" duration-ms="298" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:contactTest.createContactWithCampaign@3a52dba3]" started-at="2025-03-29T20:50:00 IST" name="afterTest" finished-at="2025-03-29T20:50:00 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method is-config="true" signature="afterSuite()[pri:0, instance:contactTest.createContactWithCampaign@3a52dba3]" started-at="2025-03-29T20:50:01 IST" name="afterSuite" finished-at="2025-03-29T20:50:01 IST" duration-ms="9" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- contactTest.createContactWithCampaign -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
